src/style.css

```diff
@@ -1,3 +1,70 @@
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background: #fff;
+}
+
+.logo {
+  display: block;
+  height: 40px;
+  width: 40px;
```

Comment:
The `.logo` class has fixed height and width values. Consider using relative units like percentages or `em` for better responsiveness across different screen sizes.